src/Build/BackEnd/Node/NodeConfiguration.cs

```diff
@@ -28,6 +28,11 @@ internal class NodeConfiguration : INodePacket
         private BuildParameters _buildParameters;
 
 #if FEATURE_APPDOMAIN
+        /// <summary>
+        /// The app domain configuration bytes sent via RPC.
+        /// </summary>
+        private byte[] _appDomainConfigBytes;
```

Comment:
Looks unused outside of `Translate`. Can it be a local variable there?

Reply:
Sure, it was remnant of the original implementation.